### DISCUSSION TITLE
Site 'Arca das Letras' - O que acharam?

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#272] [Dicas]
estava corrigindo alguns problemas de cadastro num sistema da empresa e encontrei o site [desta instituição](http://www.mda.gov.br/arcadasletras/) no sistema, é a primeira vez que encontro um site .gov bem feitinho assim, e com efeito de página de livro quando navega no menu. 

http://www.mda.gov.br/arcadasletras/

**Comment by vitocaio** [comment]
Cara, muito loko kk

**Comment by willianjusten** [comment]
Eles usam o Bookblock do Codrops, já fiz um site com isso =D
http://tympanus.net/codrops/2012/09/03/bookblock-a-content-flip-plugin/

**Comment by leticiacostadias** [comment]
Bem bacana, no celular tem alguns bugs, mas mesmo assim, muito bacana
